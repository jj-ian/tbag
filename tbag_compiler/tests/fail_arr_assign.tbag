true {
        testvar();
        endgame;
}

func int testvar() {
        int [10] i;
        i ["hello"] = 3;
        return 0;
}
