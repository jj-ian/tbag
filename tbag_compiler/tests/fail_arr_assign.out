Fatal error: exception Failure("Positional array access specifier must be an
                    Integer")
